src/components/visit/Buttons.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { useState } from 'react'
+
+import Modal from './Modal'
+
+export default function EditButton(prop: { isEdit: boolean }) {
+  if (prop.isEdit) {
+    return <CancelButton />
+  } else {
+    return (
+      <svg
+        xmlns='http://www.w3.org/2000/svg'
+        className='absolute right-0.5 bottom-0.5 h-6 w-6'
+        fill='white'
+        viewBox='0 0 24 24'
+        stroke='currentColor'
+        strokeWidth='1'
+      >
+        <path
+          strokeLinecap='round'
+          strokeLinejoin='round'
+          d='M15.232 5.232l3.536 3.536m-2.036-5.036a2.5 2.5 0 113.536 3.536L6.5 21.036H3v-3.572L16.732 3.732z'
+        />
+      </svg>
+    )
+  }
+}
+
+export const AddButton = () => {
+  const [openModal, setOpenModal] = useState(false)
+  const opening = () => {
+    if (openModal) {
+      setOpenModal(false)
+    } else {
+      setOpenModal(true)
+    }
```

Comment:
You could make use of boolean logic here to toggle the value without an if statement `const opening = () => setOpenModal(!openModal);`